pep-0008.txt

```diff
@@ -333,8 +333,8 @@ technical terms are used which aren't English). In addition, string
 literals and comments must also be in ASCII. The only exceptions are
 (a) test cases testing the non-ASCII features, and
 (b) names of authors. Authors whose names are not based on the
-latin alphabet MUST provide a latin transliteration of their
-names.
+Latin (latin-1, ISO/IEC 8859-1 character set) alphabet MUST provide 
```

Comment:
Is this fine, or move 'alphabet' before the brackets: 
"names of authors. Authors whose names are not based on the
Latin alphabet (latin-1, ISO/IEC 8859-1 character set) MUST provide 
a transliteration of their names in this character set."

Reply:
Yes, I think the parenthetical comment should come after "alphabet".

Reply:
Done.